dmscripts/notify_buyers_when_requirements_close.py

```diff
@@ -36,7 +36,7 @@ def notify_users(email_api_key, brief):
                 [user['emailAddress'] for user in brief['users'] if user['active']],
                 email_body,
                 email_api_key,
-                u'Next steps for your ‘{}’ requirements'.format(brief['title']),
+                u"Next steps for your '{}' requirements".format(brief['title']),
```

Comment:
I think 2to3 replaced the curly quotes with straight ones inside the message. Let's restore them since they're nicer.

Reply:
Haha that wasn't 2to3, that was me 😸 I guess if we're reinstating the `u""` it doesn't matter whether they're curly or not - will change back.